src/main/resources/application.properties

```diff
@@ -1 +1,29 @@
+spring.application.name=cms
+server.servlet.context-path=/cms
 
+##DATABSE
+spring.datasource.url=jdbc:h2:mem:cms_db
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=poliscrypts
+spring.datasource.password=poliscrypts*2022
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.h2.console.enabled=true
+
+##FLYWAY MIGRATION
```

Comment:
you just missed a property i think
spring.jpa.hibernate.ddl-auto = ?????
may you need to validate what you flyway generate for you :)

Reply:
`validate` , done 😉